test/automated/api/009_configmanagement.test.js

```diff
@@ -166,7 +167,10 @@ test('verify default admin configuration', async (done) => {
 	expect(res.body.yp.enabled).toBe(defaultYPConfig.enabled);
 	// expect(res.body.yp.instanceUrl).toBe(defaultYPConfig.instanceUrl);
 
-	expect(res.body.adminPassword).toBe(defaultAdminPassword);
+	// expect(res.body.adminPassword).toBe(defaultAdminPassword);
```

Comment:
I don't remember why some of these other assertions are commented out, but since the old admin password ones are no longer valid, they should probably just be removed completely.